siriuspy/tests/csdevice/test_pwrsupply.py

```diff
@@ -185,23 +182,6 @@ def test_ps_propty_database(self):
                 if propty in current_pvs:
                     self.assertEqual(dbi['unit'], unit)
 
-    def test_pu_propty_database(self):
-        """Test pu_propty_database."""
-        current_pvs = TestPwrSupply.pu_alarm
-        for pstype in TestPwrSupply.pstypes:
-            db = pwrsupply.get_pu_propty_database(pstype)
-            unit = db['Voltage-SP']['unit']
-            for propty, dbi in db.items():
-                # set setpoint limits in database
-                if propty in TestPwrSupply.ps_alarm:
-                    self.assertLessEqual(dbi['lolo'], dbi['low'])
-                    self.assertLessEqual(dbi['low'], dbi['lolim'])
-                    self.assertLessEqual(dbi['lolim'], dbi['hilim'])
-                    self.assertLessEqual(dbi['hilim'], dbi['high'])
-                    self.assertLessEqual(dbi['high'], dbi['hihi'])
-                if propty in current_pvs:
-                    self.assertEqual(dbi['unit'], unit)
-
     def test_li_ma_propty_database(self):
```

Comment:
isn't it necessary to remove this test as well?

Reply:
yes! I have removed this innocuous test. It was doing nothing since the beginning.